lib/SILOptimizer/Utils/InstOptUtils.cpp

```diff
@@ -1449,7 +1449,8 @@ static bool keepArgsOfPartialApplyAlive(PartialApplyInst *pai,
 
   ValueLifetimeAnalysis::Frontier partialApplyFrontier;
   if (!vla.computeFrontier(partialApplyFrontier,
-                           ValueLifetimeAnalysis::DontModifyCFG)) {
+                           ValueLifetimeAnalysis::DontModifyCFG) ||
+      partialApplyFrontier.empty()) {
```

Comment:
If `partialApplyFrontier` is empty (and we don't return), we will create an `alloc_stack` and `copy_addr` but no `dealloc_stack`.